### PR TITLE
python3's map() returns a map object, but we expect sources to be a list

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: src/tito/builder/main.py [hunk 1]
```diff
@@ -614,7 +614,7 @@ def tgz(self):
         debug("Scanning for sources.")
         cmd = "/usr/bin/spectool --list-files '%s' | awk '{print $2}' |xargs -l1 --no-run-if-empty basename " % self.spec_file
         result = run_command(cmd)
-        self.sources = map(lambda x: os.path.join(self.rpmbuild_gitcopy, x), result.split("\n"))
+        self.sources = list(map(lambda x: os.path.join(self.rpmbuild_gitcopy, x), result.split("\n")))
```
Comment (stevekuznetsov) [inline src/tito/builder/main.py]: Could we maybe use fewer nifty syntax here for clarity?

```python
self.sources = []
for file in result.split("\n"):
    self.sources.append(os.path.join(self.rpmbuild_gitcopy, file))
```

Comment (evgeni) [inline src/tito/builder/main.py]: Surely could, I was going after the smallest change. But if you want more readable code, that certainly can be done ;)

Comment (stevekuznetsov) [inline src/tito/builder/main.py]: If you don't mind, I think that would be nice!

Comment (evgeni) [inline src/tito/builder/main.py]: done in caa7c77

**Comment by stevekuznetsov** [comment]
@evgeni thanks for the contribution! :)

**Comment by dgoodwin** [comment]
Thanks @evgeni! and @stevekuznetsov for merging!

**Pull request merged**